kiali-server/values.yaml

```diff
@@ -30,6 +30,7 @@ clustering:
   clusters: []
 
 deployment:
+  extra_labels: {}
```

Comment:
We sort all values in these lists to make it easy to find them and just for sanitary reasons (it looks nice) :)

Move the "extra_labels" above host_aliases.

Reply:
see https://github.com/kiali/kiali-operator/blob/master/DEVELOPING.adoc#are-you-altering-a-kiali-server-configuration-setting

`Ensure each group of settings are alphabetically sorted in all the files where the new setting was added.`

Reply:
Since I got your attention, it's expected the camelCase and snake_case difference between the charts?

Reply:
Yes. You did it correctly. That snake_case is a legacy thing leftover from the old operator.

We may, in fact, be converting to camelCase in the near future, if/when [we release a new v1 of the CRD](https://github.com/kiali/kiali/issues/8237) -- but don't worry about that for now. You did it correctly.

This PR looks complete (I didn't test it yet). Just need to do similar thing over in the operator.